### ISSUE TITLE
Add g:racer_insert_paren to the documentation

### ISSUE THREAD

**Issue by pappasam** [racer-rust/vim-racer#127]
Tl;dr auto-inserting a parenthesis for functions messes up my workflow. Setting g:racer_insert_paren = 0 overrides this behavior. This isn't documented in the README and should be.

I use https://github.com/jiangmiao/auto-pairs for completing my (), {}, and []. The auto insertion of parentheses didn't work nicely with auto-pairs and I was tolerating the workflow until I read this plugin's source code. It would be nice if we could document this option in the README.

**Moreover, g:racer_insert_paren = 0 should probably default to 0**. This would make vim-racer's behavior consistent with other auto-completion Plugins I've used (https://github.com/davidhalter/jedi-vim , https://github.com/ternjs/tern_for_vim , etc)
